docs/querying/sql-functions.md

```diff
@@ -1282,21 +1282,74 @@ Returns the following:
 
 ## BLOOM_FILTER
 
-Computes a Bloom filter from values produced by the specified expression.
+Computes a [bloom filter](../development/extensions-core/bloom-filter.md) from values provided in an expression.
```

Comment:
Looks like Bloom is a name so all instances of this should be capitalized

Reply:
Fixed